samples/hello-js/build.gradle.kts

```diff
@@ -48,6 +48,49 @@ tasks {
         }
         into("$buildDir/web")
     }
+
+    val populateNodeModules by registering {
+        group = "build"
+        description = "Populate Node modules directory for karma"
+        val outputDir = file("$buildDir/$name")
+        inputs.property("testCompileClasspath", configurations.testCompileClasspath.get())
+        outputs.dir(outputDir)
+        doLast {
+            val fromJars = configurations.testCompileClasspath.get()
+                .filter { it.name.matches(Regex(".*\\.jar")) }
+                .map { zipTree(it) }
+            copy {
+                includeEmptyDirs = false
+                from(fromJars)
+                into(outputDir)
+                include("**/*.js")
+                exclude("META-INF/**")
+            }
+        }
+    }
+
+    node {
```

Comment:
:x: iirc, this is the `node` project extension, not a task. I would move its configuration out of the `tasks {}` block to make this clear. Or, if setting the node version isn't required for this sample it could simply be removed, your call.